src/MDOP.jl

```diff
@@ -4,16 +4,150 @@ using Distances
 
 export beta_statistic
 export MDOP
+export estimate_maximum_delay
 
 
 """
-    MDOP(....)
-method by [^Nichkawde2013]
+    MDOP(s::Vector; kwargs...) → Y, τ_vals, ts_vals, FNNs [,βS]
+MDOP is a unified approach to properly embed a time series (`Vector` type) or a
+set of time series (`Dataset` type) based on the paper of Chetan Nichkawde
+[^Nichkawde2013].
 
+## Keyword arguments
+
+* `τs= 0:50`: Considered delay values `τs` (in sampling time units). For each of
+  the `τs`'s the β-statistic gets computed.
+* `w::Int = 1`: Theiler window (neighbors in time with index `w` close to the point,
+  that are excluded from being true neighbors). `w=0` means to exclude only the
+  point itself, and no temporal neighbors.
+* `fnn_thres::Real= 0.05`: A threshold value defining a sufficiently small fraction
+  of false nearest neighbors, in order to the let algorithm terminate and stop
+  the embedding procedure (`0 <= fnn_thres < 1).
+* `r::Real = 2`: The threshold for the tolerable relative increase of the distance
+  between the nearest neighbors, when increasing the embedding dimension.
+* `β_condition::Bool = false`: When set `true`, the function also returnes the `β`-statistics
+  of all embedding cycles.
```

Comment:
I don't think this keyword is necessary, because `βs` are computed anyways. The function already has at least 3 return values, let's just always return `βs` as well. It is uncommon that the **value** of an argument decides the **number** of returned elements. (it is also type unstable).